<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EPKZKARW0363" ignored="0" name="testDivision" tests="16" failures="2" timestamp="30 Oct 2017 08:02:44 GMT" time="0.051" errors="0">
  <testcase name="divTest" time="0.0" classname="com.epam.unittesting.tests.DivCalculatorTest"/>
  <testcase name="divTest" time="0.0" classname="com.epam.unittesting.tests.DivCalculatorTest"/>
  <testcase name="divTest" time="0.0" classname="com.epam.unittesting.tests.DivCalculatorTest"/>
  <testcase name="divTest" time="0.0" classname="com.epam.unittesting.tests.DivCalculatorTest"/>
  <testcase name="divTest" time="0.0" classname="com.epam.unittesting.tests.DivCalculatorTest"/>
  <testcase name="divTest" time="0.006" classname="com.epam.unittesting.tests.DivCalculatorTest"/>
  <testcase name="divTest" time="0.005" classname="com.epam.unittesting.tests.DivCalculatorTest"/>
  <testcase name="divTest" time="0.0" classname="com.epam.unittesting.tests.DivDoubleCalculatorTest"/>
  <testcase name="divTest" time="0.0" classname="com.epam.unittesting.tests.DivDoubleCalculatorTest"/>
  <testcase name="divTest" time="0.0" classname="com.epam.unittesting.tests.DivDoubleCalculatorTest"/>
  <testcase name="divTest" time="0.0" classname="com.epam.unittesting.tests.DivDoubleCalculatorTest"/>
  <testcase name="divTest" time="0.0" classname="com.epam.unittesting.tests.DivDoubleCalculatorTest"/>
  <testcase name="divTest" time="0.0" classname="com.epam.unittesting.tests.DivDoubleCalculatorTest"/>
  <testcase name="divTest" time="0.0" classname="com.epam.unittesting.tests.DivDoubleCalculatorTest"/>
  <testcase name="divTest" time="0.001" classname="com.epam.unittesting.tests.DivDoubleCalculatorTest">
    <failure type="java.lang.AssertionError" message="Invalid result of division:  expected [0.0] but found [Infinity]">
      <![CDATA[java.lang.AssertionError: Invalid result of division:  expected [0.0] but found [Infinity]
	at com.epam.unittesting.tests.DivDoubleCalculatorTest.divTest(DivDoubleCalculatorTest.java:15)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- divTest -->
  <testcase name="divTest" time="0.0" classname="com.epam.unittesting.tests.DivDoubleCalculatorTest">
    <failure type="java.lang.AssertionError" message="Invalid result of division:  expected [0.0] but found [NaN]">
      <![CDATA[java.lang.AssertionError: Invalid result of division:  expected [0.0] but found [NaN]
	at com.epam.unittesting.tests.DivDoubleCalculatorTest.divTest(DivDoubleCalculatorTest.java:15)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- divTest -->
</testsuite> <!-- testDivision -->
